ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java

```diff
@@ -203,27 +202,30 @@ public void validateCredentials() {
             if (currentAccount.isPresent()) {
 
                 String service = currentAccount.get().getService();
-                Credentials current = Credentials.valueOf(service.toUpperCase());
-
-                switch (current) {
-                    case DROPBOX:
-                        service = "DropBox";
-                        break;
-                    case SALESFORCE:
-                        service = "Salesforce";
-                        break;
-                    case TWITTER:
-                        service = "Twitter";
-                        break;
-                    case S3:
-                        return; //TODO: skip for now
-                    case SLACK:
-                        service = "Slack";
-                        break;
-                    default:
-                        return; //skip for other cred
+                if (!service.equalsIgnoreCase("Google Mail")) {
+                    Credentials current = Credentials.valueOf(service.toUpperCase());
+
+                    switch (current) {
+                        case DROPBOX:
+                            service = "DropBox";
+                            break;
+                        case SALESFORCE:
+                            service = "Salesforce";
+                            break;
+                        case TWITTER:
+                            service = "Twitter";
+                            break;
+                        case S3:
+                            return; //TODO: skip for now
+                        case SLACK:
+                            service = "Slack";
+                            break;
+                        default:
+                            return; //skip for other cred
+                    }
+                } else {
+                    GMailUtils.createAccessToken();
```

Comment:
wouldn't it make more sense to add google mail to enum and call createAccessToken in case GOOGLE_MAIL?

Reply:
or is this createAccessToken call necessary here? the method is calling createConnections(accountsTalbe) (<- typo) and the createAccessToken is called again in createConnections?